packages/react/src/collapsible/trigger/CollapsibleTriggerDataAttributes.ts

```diff
@@ -0,0 +1,6 @@
+export enum CollapsibleTriggerDataAttributes {
+  /**
+   * Indicates whether the collapsible panel is opened.
+   */
+  panelOpen = 'data-panel-open',
+}
```

Comment:
Should this be imported by the actual component code?

Reply:
We could replace the actual code instead of hard-coding this to use the enum. I haven't updated the code as part of the PR tough (btw, this is why I didn't include the brackets in the value.

Reply:
Makes sense